### ISSUE TITLE
Error downloading certain libs when creating new project

### ISSUE THREAD

**Issue by activestylus** [tower-archive/tower#213]
Some of the install links are 404ing

```
Error downloading https://raw.github.com/viatropos/tower/master/dist/tower.js

Error downloading https://raw.github.com/logicalparadox/chai/master/chai.js
```

**Comment by lancejpollard** [comment]
This looks like it's old code, haven't yet pushed master to npm.

[packages/tower-generator/server/generators/tower/app/appGenerator.coffee#L135](https://github.com/viatropos/tower/blob/0281496c802930d8e838c377210266fc5b7c8b11/packages/tower-generator/server/generators/tower/app/appGenerator.coffee#L135)

**Comment by lancejpollard** [comment]
Fixed, 0.4.1 is pushed to npm.

**Comment by activestylus** [comment]
Sweet
